tracer/test/Datadog.Trace.Security.IntegrationTests/IAST/Deduplicaation/DeduplicationTests.cs

```diff
@@ -0,0 +1,54 @@
+// <copyright file="DeduplicationTests.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+#if NETCOREAPP
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+using Datadog.Trace.TestHelpers;
+using VerifyXunit;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Datadog.Trace.Security.IntegrationTests.Iast;
+
+[UsesVerify]
+public class DeduplicationTests : TestHelper
+{
+    private const string ExpectedOperationName = "weak_hashing";
+    private static readonly Regex LocationMsgRegex = new(@"(\S)*""location"": {(\r|\n){1,2}(.*(\r|\n){1,2}){0,3}(\s)*},");
+
+    public DeduplicationTests(ITestOutputHelper output)
+        : base("Deduplication", output)
+    {
+        SetServiceVersion("1.0.0");
+    }
+
+    [SkippableTheory]
+    [Trait("Category", "EndToEnd")]
+    [Trait("RunOnWindows", "True")]
+    [InlineData(false)]
+    [InlineData(true)]
+    public async Task SubmitsTraces(bool deduplicationEnabled)
+    {
+        SetEnvironmentVariable("DD_IAST_ENABLED", "true");
+        SetEnvironmentVariable("DD_IAST_DEDUPLICATION_ENABLED", deduplicationEnabled.ToString());
+
+        int expectedSpanCount = deduplicationEnabled ? 1 : 5;
+        var filename = deduplicationEnabled ? "deduplication.deduplicated" : "deduplication.duplicated";
```

Comment:
maybe add the iast.prefix to identify more easily the tests related to the snapshots

Reply:
Done!